kubernetes/AKS/README.md

```diff
@@ -0,0 +1,262 @@
+
+```az aks install-cli```
+
+To configure `kubectl` to connect to your Kubernetes cluster, use the [az aks get-credentials](https://docs.microsoft.com/en-us/cli/azure/aks?view=azure-cli-latest#az-aks-get-credentials) command. This command downloads credentials and configures the Kubernetes CLI to use them.
+
+```az aks get-credentials --resource-group myResourceGroup --name myAKSCluster```
+
+#### 1.5 Install NVIDIA device plugin
+
+Before the GPUs in the nodes can be used, you must deploy a DaemonSet for the NVIDIA device plugin. This DaemonSet runs a pod on each node to provide the required drivers for the GPUs.
+
+```kubectl apply -f nvidia-device-plugin-ds.yaml```
+
+NAME                  READY  STATUS  RESTARTS  AGE
+
+nvidia-device-plugin-daemonset-7lvxd  1/1   Running  0     42s
+
+#### 1.6 Install helm
+
+```
+curl -fsSL -o get_helm.sh https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3
+chmod 700 get_helm.sh
+./get_helm.sh
+```
+
+### 2、Deploy TorchServe on AKS
+
+#### 2.1 Download the github repository and enter the kubernetes directory
+
+```git clone https://github.com/pytorch/serve.git```
+
+```cd serve/kubernetes/AKS```
+
+#### 2.2 Create a storage class
+
+A storage class is used to define how an Azure file share is created. If multiple pods need concurrent access to the same storage volume, you need Azure Files. Create the storage class with the following kubectl apply command:
+
+```kubectl apply -f templates/Azure_file_sc.yaml```
+
+#### 2.3 Create PersistentVolume
+
+```kubectl apply -f templates/AKS_pv_claim.yaml```
+
+Your output should look similar to
+
+persistentvolumeclaim/model-store-claim created
+
+Verify that the PVC / PV is created by excuting.
+
+```kubectl get pvc,pv```
+
+Your output should look similar to
+
+NAME                                      STATUS   VOLUME                                     CAPACITY   ACCESS MODES   STORAGECLASS      AGE
+persistentvolumeclaim/model-store-claim   Bound    pvc-c9e235a8-ca2b-4d04-8f25-8262de1bb915   1Gi        RWO            managed-premium   29s
+
+NAME                                                        CAPACITY   ACCESS MODES   RECLAIM POLICY   STATUS   CLAIM                       STORAGECLASS      REASON   AGE
+persistentvolume/pvc-c9e235a8-ca2b-4d04-8f25-8262de1bb915   1Gi        RWO            Delete           Bound    default/model-store-claim   managed-premium            28s
+
+#### 2.4 Create a pod and copy MAR / config files
+
+Create a pod named `pod/model-store-pod` with PersistentVolume mounted so that we can copy the MAR / config files.
+
+```kubectl apply -f templates/model-store-pod.yaml```
```

Comment:
```suggestion
```kubectl apply -f templates/model_store_pod.yaml```
```
The checked in yaml file uses underscores instead